packages/drivers/driver-web-cache/src/FluidCacheIndexedDb.ts

```diff
@@ -13,14 +13,15 @@ import { FluidCacheErrorEvent } from "./fluidCacheTelemetry";
 export const FluidDriverCacheDBName = "fluidDriverCache";
 
 // The name of the object store within the indexed db instance that the driver will use to cache Fluid content.
-export const FluidDriverObjectStoreName = "driverStorage.V3";
+export const FluidDriverObjectStoreName = "driverStorage.V4";
```

Comment:
Do we have to change version? It feels like we can keep using old schema, and maybe even write lastAccessTimeMs out, in case code rollback happens and old code would read these entries. In the next minor release (which would be after existing policy of 3 days for cache expiration) we can stop writing this property.

Reply:
I like this suggestion, where we can keep everything same in the schema and just write the last access time when putting value in cache and not re update it when reading an item from the cache.